.github/workflows/sync-to-staging.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync release to staging
+
+on:
+  release:
+    types: [prereleased, released]
```

Comment:
There's a note in the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release):

> Note: The prereleased type will not trigger for pre-releases published from draft releases, but the published type will trigger. If you want a workflow to run when stable and pre-releases publish, subscribe to published instead of released and prereleased.

I think that means the types here should be switched to `[published]`.